src/unstract/llmwhisperer/client.py

```diff
@@ -282,7 +285,7 @@ def generate():
             "whisper_hash": response.headers["whisper-hash"],
         }
 
-    def whisper_status(self, whisper_hash: str) -> dict:
+    def whisper_status(self, whisper_hash: str, encoding: str = "utf-8") -> dict:
```

Comment:
@ShoaibMajidDar I don't think we require encoding to be added to `whisper_status()` since we usually respond with a status message only that's generally UTF-8. However we'll have to add it to `whisper_retrieve()` since we use that in case of async calls to obtain the extracted text which might contain special characters

Reply:
I have resolved the issue